com.unity.render-pipelines.universal/Runtime/Passes/AdditionalLightsShadowCasterPass.cs

```diff
@@ -20,8 +20,13 @@ private static class AdditionalShadowsConstantBuffer
             public static int _AdditionalShadowmapSize;
         }
 
+        [Obsolete("AdditionalLightsShadowCasterPass.m_AdditionalShadowsBufferId was deprecated. Shadow slice matrix is now passed to the GPU using an entry in buffer m_AdditionalLightsWorldToShadow_SSBO", false)]
         public static int m_AdditionalShadowsBufferId;
+        [Obsolete("AdditionalLightsShadowCasterPass.m_AdditionalShadowsIndicesId was deprecated. Shadow slice index is now passed to the GPU using last member of an entry in buffer m_AdditionalShadowParams_SSBO", false)]
         public static int m_AdditionalShadowsIndicesId;
```

Comment:
We've recently started moving deprecated things into one of:
- Editor/Deprecated.cs
- Runtime/Deprecated.cs
- Deprecated.hlsl

Depending on where that code lives.

For the .cs files we do so by making the class partial, and creating another partial partition in the Deprecated file. This would make sense to move there as well. Then in 11.x we will change this to [Obsolete(""), true] and remove it in 12.x.

Reply:
Thanks for the explanations.
I moved this deprecated variable declaration to the Deprecated.cs file in commit dabaf72